tests/unit/insertTestResult.unitTest.ts

```diff
@@ -1223,6 +1223,78 @@ describe("insertTestResult", () => {
         });
     });
 
+    context("when inserting a non-adr HGV with null expiry Date and null certificateNumber", () => {
+        it("should not throw error", () => {
+            const testResult = {...testResultsPostMock[4]};
```

Comment:
suggest to use cloneDeep instead of spread operator.

Reply:
fixed